config/routes.rb

```diff
@@ -17,8 +17,11 @@
 
   namespace :api do
     namespace :v1 do
-      resources :recipes
+      resources :recipes do
+        resources :comments
+      end
       resources :users
+      resources :comments
```

Comment:
I see you have this in here twice. I think it made sense for testing, but we probably just want the one nested under recipes?